API Guideline - Return list of API resources under a `results` array (#138)
meilisearch/specifications: text/0132-indexes-api.md

|--------------------------|--------------------------|----------|
| `offset`                 | integer / `null`         | false    |
| `limit`                  | integer / `null`         | false    |

###### 3.2.1.1.1. `offset`

- Type: Integer
- Required: False
- Default: `0`

Sets the starting point in the results, effectively skipping over a given number of indexes.

###### 3.2.1.1.2. `limit`

- Type: Integer
- Required: False
- Default: `20`

Sets the maximum number of indexes to be returned by the current request.

##### 3.2.1.2. Response Definition

An object containing all the indexes.

| Field                    | Type                     | Required |
|--------------------------|--------------------------|----------|
| `results`                | Array[Index]             | true     |
| `offset`                 | integer                  | true     |
| `limit`                  | integer                  | true     |
| `total`                  | integer                  | true     |

###### 3.2.1.2.1. `results`

- Type: Array[Index]
- Required: True

An array containing the fetched indexes.

###### 3.2.1.2.2. `offset`

- Type: Integer
- Required: True

> Gives the `offset` parameter used for the query.

Reviewer: ```suggestion
The `offset` contains the number of offset documents.
```

> See [3.2.1.1.1. `offset`](#32111-offset) section.

###### 3.2.1.2.3. `limit`

- Type: Integer
- Required: True

> Gives the `limit` parameter used for the query.

Reviewer: ```suggestion
The `limit` parameter contains the max number of indexes returned in `results`.
```

> See [3.2.1.1.2. `limit`](#32112-limit) section.

###### 3.2.1.2.3. `total`

- Type: Integer
- Required: True

> Gives the total number of indexes that can be browsed.

Reviewer: ```suggestion
The `total` contains the total number of indexes that can be browsed.
```
